tests/php/src/Validation/ScannableURLProviderTest.php

```diff
@@ -425,9 +426,21 @@ public function test_get_taxonomy_links() {
 				);
 			}
 
+			$post = self::factory()->post->create();
+
+			// `wp_pattern_category` is a special taxonomy that is registered for categorizing patterns.
+			// It is only associated with the `wp_block` post type, so we need to create a `wp_block` post.
+			if ( 'wp_pattern_category' === $taxonomy ) {
+				$post = self::factory()->post->create(
```

Comment:
This post is overriding the `$post` created just before. Should the above `$post` creation be put in an `else`?

Reply:
Done in https://github.com/ampproject/amp-wp/pull/7613/commits/9b700fffab2bc73a3c19cd9dce10777c23337d61